specs/span-batches.md

```diff
@@ -220,7 +222,7 @@ The assumption makes upper inequality to hold. Therefore, we decided to manage `
   - If the last block references canonical L1 chain as its origin,
     we can ensure the all other blocks' origins are consistent with the canonical L1 chain.
 - Parent hash
-  - In V0 Batch spec, we need batch's parent hash to validate if batch's parent is consistent with current L2 safe head.
+  - In V0 Batch spec, we need batch's parent hash to validate if batch's parent is consistent with current L2 chain.
```

Comment:
Maybe mention "safe portion of the chain". I guess it wouldn't really happen that we would be asking for a span batch that started at some timestamp where it started at some point in the unsafe chain.

Reply:
This seems to be my mistake. Since this sentence is about batch v0 (singular batch), it would be correct to say `safe head`.